test/indexer/__init__.py

```diff
@@ -92,6 +92,7 @@ def _load_canned_result(self, bundle_fqid: BundleFQID) -> MutableJSONs:
             hit['_index'] = config.es_index_name(catalog=self.catalog,
                                                  entity_type=index_name.entity_type,
                                                  aggregate=index_name.aggregate)
+            hit.pop('sort')
```

Comment:
This and/or the removal of `sort` in `_get_all_hits()` could use a short comment explaining why we need to remove this field for our test.